test/crash_test/src/crash_test.c

```diff
@@ -74,6 +94,6 @@ crash_test_init(void)
     shell_cmd_register(&crash_cmd_struct);
 #endif
 #if MYNEWT_VAL(CRASH_TEST_NEWTMGR)
-    mgmt_group_register(&crash_test_nmgr_group);
+    mgmt_register_group(&crash_test_nmgr_group);
```

Comment:
Can you change `crash_test_nmgr_handler` -> `crash_test_mgmt_handler`, I think that would be the right name since we are removing newtmgr.